teams.md

```diff
@@ -0,0 +1,208 @@
+* Build and Distribute Releases of the _pandas_ library
+
+#### Permissions
+
+* Has permission to merge anything to the main branch of the _pandas _repository
+* Only Team with PDEP voting rights
+
+#### Criteria for Membership
+
+* New Members are nominated after a period of sustained and qualitative contributions
+    * Joining heuristic: "is there some aspect of _pandas _where we trust this person to
+      get something in without anyone's help?".
+    * An existing Member of the Team proposes to promote a new member
+* [https://pandas.pydata.org/docs/dev/development/maintaining.html#becoming-a-pandas-maintainer](https://pandas.pydata.org/docs/dev/development/maintaining.html#becoming-a-pandas-maintainer)
+
+### pandas-stubs Team
+
+#### Responsibilities
+
+* Maintenance of `pandas-stubs` repository
+  ([https://github.com/pandas-dev/pandas-stubs](https://github.com/pandas-dev/pandas-stubs)) 
+    * Triage of issues
+    * Reviews of pull requests
+    * Merges
+    * Releases
+        * Delegated to 2 or 3 members
+
+#### Permissions
+
+* Only Team with permission to merge to the main branch of the _pandas-stubs_ repository
+
+#### Criteria for Membership
+
+* New Members are nominated after a period of sustained and qualitative contributions
+    * Joining heuristic: "is there some aspect of _pandas-stubs _where we trust this
+      person to get something in without anyone's help?".
+    * An existing Member of the Team proposes to promote a new member
+
+### Finance Team
+
+#### Responsibilities
+
+* Manage Applications for Grant Proposals for Funding
+* Approve Project Expenses
+* Ensure that finances are effectively used
+* Find funding opportunities from appropriate, ethical sources 
+
+#### Permissions
+
+* No specific technical permissions
+* Permissions to make all financial decisions on behalf of the Project
+* Determines rates for any paid contributors
+
+#### Criteria for Membership
+
+* Member of one Team for at least three years
+* Unanimous Approval of Steering Committee
+* Limited to 5 Members
+
+### Infrastructure Team
+
+#### Responsibilities
+
+* Keep the pandas infrastructure up and working, including the servers for the website,
+  benchmarks, CI and others as needed.
+* Decisions on platform support
+* Manage the 1Password repository
+
+#### Permissions
+
+* Usernames and Passwords for various servers
+
+#### Criteria for Membership
+
+* New Members are nominated after a period of sustained and qualitative contributions
+    * Joining heuristic: "is there some aspect of the infrastructure where we trust this
+      person to manage that part of the infrastructure without assistance?".
+    * An existing Member of the Team proposes to promote a new member
+
+### Documentation Team
+
+#### Responsibilities
+
+* Maintain the pandas documentation and its (building) infrastructure
```

Comment:
is there some overlap here with infrastructure team?

Reply:
That depends if we are having the infrastructure team have build responsibilities. We put the word "(building)" in there to refer to the Sphinx building process.

Reply:
clarified that this is about Sphinx in next revision